homeassistant/components/scrape/sensor.py

```diff
@@ -118,26 +118,31 @@ def state(self):
         """Return the state of the device."""
         return self._state
 
+    def _extract_value(self):
+        """Parse the html extraction in the executor."""
+        raw_data = BeautifulSoup(self.rest.data, "html.parser")
```

Comment:
Is this blocking?

Reply:
<img width="1338" alt="Screen Shot 2020-10-17 at 10 06 24 AM" src="https://user-images.githubusercontent.com/663432/96346286-7aea1c80-1060-11eb-8c77-0622defb9368.png">

It will block the event loop long enough that I noticed when I added 10 of these for testing.